zuora.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rspec'
   s.add_development_dependency 'simplecov'
   s.add_development_dependency 'test-unit'
+  s.add_development_dependency 'timecop'
   s.add_development_dependency 'yard'
   s.add_dependency 'akami'
   s.add_dependency 'httpi'
```

Comment:
Can we remove `httpi` as a dependency please? That is a dependency of `savon` < 3.

Can we change the `savon` dependency to guard against using v3.0.0 when it's released? That version replaces `httpi` with `faraday` and contains breaking changes.

``` ruby
s.add_dependency 'savon', '< 3'
```

Reply:
Addressed in this commit [a8aa20d](https://github.com/Sage/zuora/pull/31/commits/a8aa20d22b037aec5533aa8a090e532e68b0ea25)